src/turn_integrator.py

```diff
@@ -125,16 +227,17 @@ def destroy_claim(msisdn, line_name, claim_uuid):
     print(response.text)
     return response
 
-### Journey management
-### https://whatsapp.turn.io/docs/api/stacks
+### JOURNEYS
+## Start a journey for a specific user. Details here:
+## https://whatsapp.turn.io/docs/api/stacks
 
-def start_journey(msisdn, line_name, stack_uuid):
-    journey_data = {"wa_id": msisdn}
+def start_journey(msisdn: str, line_name: str, stack_uuid: str) -> requests.Response:
```

Comment:
@DylanIvanBrown We do have the profile update method, which we can use for updating contact vars, so one could just update the profile with whatever, and then start a new stack. I _think_ that would be sufficient?

Reply:
My only concern with updating separately would be pseudo-race conditions where the change we want to make isn't reflected as fast as the journey starts, and then some incorrect decision is made based on that. This would be a tricky bug to track down if it happens.